packages/pluggableWidgets/progress-circle-web/src/ProgressCircle.editorPreview.tsx

```diff
@@ -0,0 +1,56 @@
+import { createElement, ReactElement } from "react";
+import { ProgressCirclePreviewProps } from "../typings/ProgressCircleProps";
+import { ProgressCircle } from "./components/ProgressCircle";
+import { parseStyle } from "@widgets-resources/piw-utils";
+import { defaultValues, ProgressCircleValues } from "./progressCircleValues";
+import { calculatePercentage } from "./util";
+
+export function preview(props: ProgressCirclePreviewProps): ReactElement {
+    function getProgressBarValues(): ProgressCircleValues {
+        switch (props.type) {
```

Comment:
I'd prefer to see this as a local argument to the function, i.s.o. higher scoped, however this is largely preference, so this is not a change-request, unless you agree.